### PR TITLE
226 insights metadata improvements

### PR DESCRIPTION
**Pull request by elevchyt**
Improves metadata of insights cards based on available info.

### CODE REVIEW

File: apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ export default function InsightsGrid(props: PropsType): ReactNode {
                   description={insight.position}
```
Comment (trixobird) [inline apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx:35]: in the line above: `title={insight.adName ?? insight.publisher ?? ''}` what if there is neither adName nor publisher?

**Review by trixobird — APPROVED** [review]
one minor thing, feel free to merge after fix

File: apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx (hunk 1 above)
Comment (elevchyt) [reply apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx]: will fallback to a generic title now

File: apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx [hunk 2]
```diff
@@ -29,10 +29,12 @@ export default function InsightsGrid(props: PropsType): ReactNode {
           ? props.insights.map((insight) => (
               <InsightCard
                 key={insight.id as Key}
-                title={insight.adName ? insight.adName : insight.publisher}
+                // eslint-disable-next-line no-nested-ternary -- cleaner than writing an entire function
+                title={insight.adName ? insight.adName : insight.publisher ? insight.publisher : t('insight')}
```
Comment (trixobird) [inline apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx:33]:
```suggestion
                title={insight.adName ?? insight.publisher ?? t('insight')}
```
isn't this better?

**Review by trixobird — APPROVED** [review]

File: apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx (hunk 2 above)
Comment (elevchyt) [reply apps/web/src/app/(authenticated)/insights/components/insights-grid.tsx:33]: Yup!

**Pull request merged**